examples/3_Liu_Copper_OrgSyn/example_liu.ipynb

```diff
@@ -772,54 +791,54 @@
     "## Product and characterization\n",
     "outcome = reaction.outcomes.add()\n",
     "outcome.reaction_time.CopyFrom(unit_resolver.resolve('6 h'))\n",
-    "analysis = reaction_pb2.ReactionAnalysis() # for handy enum reference\n",
+    "analysis = reaction_pb2.Analysis() # for handy enum reference\n",
     "\n",
     "outcome.analyses['isolated_weight'].type = analysis.WEIGHT\n",
+    "outcome.analyses['isolated_weight'].is_of_isolated_species = True\n",
     "outcome.analyses['isolated_weight'].details = '27.5-28.0 g recovered after workup'\n",
```

Comment:
This is interesting ("outcome.analyses['isolated_weight'].details = '27.5-28.0 g recovered after workup"). Do we need better structure in `Analysis` to allow for e.g. a `FloatValue` field?

Reply:
That relates to our earlier conversations about how much structure we want to give analytical results. If we add a FloatValue to analysis, we should also add a Temperature (for melting point), maybe Mass (since this example requires knowing the units, too), etc.

Another way to record this information would be inside the `outcome.analyses['isolated_weight'].data` map I suppose

Reply:
I think I want to avoid using the Data message for this, since we don't have `precision` there.

I'm not sure I want to go down the road of adding additional structured value messages here yet; I think we decided to punt on that for now. We do have a quantitative yield value in the linked ProductMeasurement.

WDYT?

Reply:
Agreed. The quantitative yield seems sufficient here. There's also the product's `amount` field which has a mass of 27.75 g recorded at the moment; in principle, that should probably have an associated precision of 0.25 g or so